llap-server/src/java/org/apache/hadoop/hive/llap/cache/LowLevelLrfuCachePolicy.java

```diff
@@ -43,6 +43,8 @@
  * Additionally, buffer locking has to be handled (locked buffer cannot be evicted).
  */
 public final class LowLevelLrfuCachePolicy implements LowLevelCachePolicy {
+  public static final int IN_LIST = -2;
```

Comment:
Similar to invalidate constants: a) why negative valeues b) why not enum?

Reply:
was there before, i can do that in a follow up.